tests/sympc/protocol/protocol_test.py

```diff
@@ -8,3 +13,15 @@ class TestProtocol(metaclass=Protocol):
 def test_register_protocol() -> None:
     assert "TestProtocol" in Protocol.registered_protocols
     assert Protocol.registered_protocols["TestProtocol"] == TestProtocol
+
+
+def test_exception_unsupported_fss_operation():
+    with pytest.raises(ValueError):
+        _generate_primitive("relu", 10)
+
+
+def test_exception_insufficient_fss_primitives():
+    store = {}
+    store["fss_eq"] = [[10]]
+    with pytest.raises(ValueError):
+        _get_primitive("fss_eq", store, 100)
```

Comment:
MAYBE: Could you use the `get_primitive` function (from the same module `sympc.protocol.fss.fss`).
The functions that start with `_` are meant to be private.